src/awesome_map/BoundaryInterceptor.js

```diff
@@ -150,6 +154,20 @@ define(function(require) {
          */
         this._easing = options.easing || EasingFunctions.easeOutQuart;
 
+        this._boundarySensitivity = options.boundarySensitivity || DEFAULT_BOUNDARY_SENSITIVITY;
+        /**
+         * Observable for subscribing to scroll events beyond AwesomeMap boundaries.
+         *
+         * @method AwesomeMap#onScrollPastTopBoundary
+         * @param {Function} callback
+         *        Invoked with (sender, {
+         *            boundary: {@link BoundaryTypes}
+         *        })
+         */
+        // @TODO: Does this need to be disposed somewhere?  If so, where?
```

Comment:
I think the pattern of disposing that we do is a little paranoid.  If BoundaryInterceptor is on an Awesome Map that disposes, it will be disconnected and get GCed.  Not that we typically ever dispose.  I say we cross that bridge when it becomes a problem, or we wind up over-engineering.